doc/releases/changelog-dev.md

```diff
@@ -109,8 +110,23 @@
   RX(-1, wires=[0])
   ```
 
+* Added the `qml.is_hermitian` and `qml.is_unitary` function checks.
+  [(#2960)](https://github.com/PennyLaneAI/pennylane/pull/2960)
+
+  ```pycon
+  >>> op = qml.PauliX(wires=0)
+  >>> qml.is_hermitian(op)
+  True
+  >>> op2 = qml.RX(0.54, wires=0)
+  >>> qml.is_hermitian(op2)
+  False
+  ```
+
 <h3>Breaking changes</h3>
 
+* Using an operator that might not be hermitian as an observable now raises a warning instead of an
+  error. [(#2960)](https://github.com/PennyLaneAI/pennylane/pull/2960)
```

Comment:
Maybe we should add a comment along the lines of "To definitively determine whether or not an Operator is Hermitian, use `qml.is_hermitian`".

Reply:
Done!